openpype/modules/ftrack/ftrack_module.py

```diff
@@ -479,6 +482,50 @@ def cli(self, click_group):
         click_group.add_command(cli_main)
 
 
+def _check_ftrack_url(url):
+    import requests
+
+    try:
+        result = requests.get(url, allow_redirects=False)
+    except requests.exceptions.RequestException:
+        return False
+
+    if (result.status_code != 200 or "FTRACK_VERSION" not in result.headers):
+        return False
+    return True
+
+
+def check_ftrack_url(url, log_errors=True, logger=None):
```

Comment:
I think `resolve_ftrack_url` is a better name maybe?

Reply:
Also `log_errors` argument is unused?

Reply:
Done